packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -210,19 +210,33 @@ export default class Meeting extends StatelessSparkPlugin {
     this.mediaId = locus.mediaId;
   }
 
-  setRemoteStreams(pc) {
+  setRemoteStream(pc) {
     // Receiving remote media streams
     pc.ontrack = (event) => {
-      [this.mediaStream, this.localMediaStream] = event.streams;
+      [this.remoteStream] = event.streams;
 
+      // TODO: It's possible for media to not be present
+      // so we might need to either
+      // A) wait until we have media flowing
+      // B) trigger a second event when video is flowing
       this.trigger('media:ready', {
         type: 'remote',
-        stream: this.mediaStream
+        stream: this.remoteStream
```

Comment:
hmmmm did all the refs get changed everywhere?

Reply:
ooo. I only changed them in this file. I'll check now

Reply:
I just looked and didn't see anything that depended on mediaStream (outside of the share.js file)